tests/unit/services/configuration.unitTest.ts

```diff
@@ -0,0 +1,32 @@
+import {AuthorizerConfig, configuration, validate} from "../../../src/services/configuration";
+
+describe("configuration()", () => {
+  const mockConfig: AuthorizerConfig = {
+    roleToResources: [
+      {
+        roleName: 'a-role',
+        associatedResources: [
+          '/a-resource/with-child'
+        ]
+      }
+    ]
+  };
+
+  it('should successfully return any default configuration', (): void => {
+    expect(configuration()).resolves.toBeDefined();
+  });
+
+  it('should successfully validate configuration if it\'s valid', (): void => {
+    expect(validate(mockConfig)).toEqual(mockConfig);
+  });
+
+  it('should fail if configuration object is null', (): void => {
+    // @ts-ignore
+    expect((): void => validate(null)).toThrowError('configuration is null or blank');
+  });
+
```

Comment:
Do we want to add a few more for the `associatedResources` and `roleName`? What do you think?

Reply:
Addressed https://github.com/dvsa/cvs-svc-authoriser/pull/30/commits/335c541d48771e867f0e8e787796353bcf0200ea